packages/library/src/commonMain/kotlin/io/realm/Realm.kt

```diff
@@ -125,9 +63,7 @@ class Realm private constructor(configuration: RealmConfiguration, dbPointer: Na
      * @param clazz The class of the objects to query for.
      * @return The result of the query as of the time of invoking this method.
      */
-    override fun <T : RealmObject> objects(clazz: KClass<T>): RealmResults<T> {
-        return super.objects(clazz)
-    }
+    override fun <T : RealmObject> objects(clazz: KClass<T>): RealmResults<T>
```

Comment:
Is there a reason we override the interface method here? Is it just for discoverability when browsing the source code or does it provide some other benefit? 

(I'm not against doing it just for discoverability, but then we should be consistent and it for all methods in BaseRealm)

Reply:
Just to be able to differentiate the docs.

Reply:
Ok 👍